lib/recharge.dart

```diff
@@ -11,12 +12,15 @@ import 'package:watcher/watcher.dart';
 class Recharge {
   final String path;
   final void Function()? onReload;
+  int delay;
```

Comment:
Lets make this Duration?

Reply:
It is up to you. I think keeping it the way it is now is okay too. Because delays are generally in millisecond range (500ms-20ms). 200ms seems like a good default value in my opinion

Reply:
I was not talking about the number. I meant to make the datatype Duration(milliseconds: 200) instead of int. It is better to use the built in datatype that is meant for this purpose.